internal/service/autoscaling/group.go

```diff
@@ -81,6 +81,10 @@ func ResourceGroup() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"desired_capacity_type": {
+				Type:     schema.TypeString,
```

Comment:
`Computed: true`
As you mentioned in the docs that there's a default value for it.